src/web/mjs/engine/Connectors.mjs

```diff
@@ -3,6 +3,7 @@ export default class Connectors {
     constructor(ipc) {
         ipc.listen('on-connector-protocol-handler', this._onConnectorProtocolHandler.bind(this));
         this._list = [];
+        this._favList = [];
```

Comment:
since im making a new window with a new list, will need a new list in the code as well.